### ISSUE TITLE
Error Layout v.1.9

### ISSUE THREAD

**Issue by alcir-junior-caju** [JoomShaper/Helix3#67]
Hello, I think this version 1.9 has some problem in the area of ​​layouts, it is only in default.

**Comment by pepperstreet** [comment]
Do you mean the template style is restricted to "default" settings? You can't change or save another setting? Or are you talking about the "settings" **select-box** inside the layout tab? I remember there were usability issues and confusions in previous version < 1.9

Would you mind to explain your **steps** and what does not work as expected?

**Comment by alcir-junior-caju** [comment]
I can create new layouts, but I can not select the layouts I created, they are only in default.

In version 1.8 I was able to create and choose the layouts in select, but when I upgraded to 1.9, this error started.

**Comment by pepperstreet** [comment]
I guess, I know what you are talking about. As far as I know, the layout save and load functionality does work. 

### Issue  

The only **issue** is the SELECTBOX display and **active element**. Visually, it seems to revert to the **first option element**. If I select a saved layout, I see the respective Rows and Columns are loaded. I can also save the Style and keep that layout. But if I open and edit the Style and go to layout tab… the select-box shows the wrong name!  

![helix3_layout_selectbox_wrong_active_element](https://cloud.githubusercontent.com/assets/1189521/22802030/905628a0-ef0f-11e6-8346-4c7a273b1c77.png)

To my knowledge, this **issue** is present since the feature has been introduced. ;)  
I just forgot to report and document it.  

### Suggestion

Actually, It should work like this:  
- Either show the currently selected, last loaded layout file.  
- Or just show a common label e.g. " - Choose layout… - "  

Currently it is very confusing.

**Comment by alcir-junior-caju** [comment]
That's exactly what happens, I had a problem that in the end I lost the configuration I created.

Would he soon have a solution for this?

**Comment by pepperstreet** [comment]
Yeah, I remember I had also problems in the past. Really annoying. I got used to it, and tried to use it very "carefully". For sure, this has to be changed/fixed.  

Maybe you should adjust the **title** of this issue. For clarification and to get better attention.

**Comment by alcir-junior-caju** [comment]
I will hide this option until it is stable, since I need to launch a product and I can not keep that error.

I checked the code and I did not understand it very well, I know that it makes an appointment to get the template and makes the json listing of the layouts, I think it should save the json configuration in the database.

But in the midst of it he ends up overwriting and the whole problem.

For now I will comment on the return of this function and with more time try to solve because it is quite nice this function.
